src/index.js

```diff
@@ -61,11 +75,14 @@ const updateState = updateGrid(cols, rows)
 
 const randomizeState = () => updateState(r => c => !!random(0, 1))
 
+// Why use get.length here? tbh I don't even know if that's 2 or 3.
```

Comment:
`get.length` above has 3 explicit arguments. This is the one line I'm least happy with, but couldn't figure out how to avoid. The use of `curry` here is basically to override the arity of `getNextCellState` which is otherwise 0, so `nextState` doesn't wait for all its arguments to evaluate.